examples/swapi/swapi.ts

```diff
@@ -0,0 +1,131 @@
+/**
+ * Typescript version clientlib for a subset of data in https://swapi.dev/
+ */
+
+import fetch from 'node-fetch';
+import * as url from 'url';
```

Comment:
I know the status quo uses the `url` import but tbh this is probably a mistake, this is easy using stdlib:

```
new URL('/people/123', 'https://swapi.dev/api/');
```

or if you don't want to rely on a magic .toString()

```
(new URL('/people/123', 'https://swapi.dev/api/')).href
```